examples/benchmarks/bubblebench/editable-shared-tree/src/bubblebench.ts

```diff
@@ -11,7 +11,7 @@ import {
 	ISharedTree,
 	SharedTreeFactory,
 	fail,
-} from "@fluidframework/tree";
+} from "@fluidframework/tree/internal";
```

Comment:
Is it okay for some examples to use internal APIs?
I think the preference is not to. We should at least separate the public from internal API use here (two imports) and in schema.ts.
Probably there should be a work item to resolve use of internal APIs.

Reply:
I just looked at the APIs in bublebench.ts and schema.ts, and they're all marked as internal. I'll create work item to exclude/resolve use of internal APIs from examples. This work item will be part of Tech Debt, right?

Reply:
I happened to know the background for this package.

This benchmark targetted the SharedTree internals while we worked to surface the public API.

A replacement benchmark that targets the public API went into PR yesterday (#19604).

Depending on the numbers, we may choose to keep the two side-by-side for a while, but the package is probably close to retirement.

Reply:
I don't know it if it is tech debt or if there are other plans. Using all and only `@internal` APIs is unfortunate.

Reply:
Seems like and internal or test path would be better (less confusing). Otherwise, put comments in the README.md (and hope people with questions look there -- I did for a while until I repeatedly didn't find any helpful notes and now just do so occassionally :) ).